dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:stack_trace/stack_trace.dart';
+
+import 'protocol/noop_origin.dart'
+    if (dart.library.html) 'protocol/origin.dart';
+import 'protocol/sentry_stack_frame.dart';
+
+class SentryStackTraceFactory {
+  const SentryStackTraceFactory();
+
+  List<SentryStackFrame> getStackFrames(dynamic stackTrace) {
+    if (stackTrace == null) return [];
```

Comment:
```suggestion
    if (stackTrace == null) return null;
```

we should always serialize something to null if its empty

Reply:
https://github.com/getsentry/sentry-java/blob/1ad9403c9c50396de895b0ead47d5f2e4ba5a0a7/sentry/src/main/java/io/sentry/SentryStackTraceFactory.java#L33

Reply:
✅